core/src/main/java/org/apache/accumulo/core/util/compaction/CompactionJobPrioritizer.java

```diff
@@ -44,6 +49,11 @@ public static short createPriority(CompactionKind kind, int totalFiles, int comp
         return (short) prio;
       case SELECTOR:
       case SYSTEM:
+        // Given that tablets with too many files cause several problems,
+        // boost their priority to the maximum allowed value.
+        if (condition == Condition.TABLET_OVER_SIZE) {
+          return Short.MAX_VALUE;
```

Comment:
Could possibly increment a metric for this condition, although I'm not sure how useful it would be. I think knowing that all of the external compactors and compaction threads in the tservers are fully utilized, along with a backlog of waiting compactions, is a more useful indication that more capacity is needed.

Reply:
In your changes to the monitor its now scanning the metadata table periodically.  In that scan it could compute the number of tablets over the max per table and display that on the monitor.